src/main/java/org/openstreetmap/atlas/geography/atlas/multi/MultiArea.java

```diff
@@ -39,25 +42,38 @@ public long getIdentifier()
         return this.identifier;
     }
 
+    public SubAreaList getSubAreas()
```

Comment:
Concerning thread safety, yes this is a valid concern. I will point out that the `MultiAtlas` was already *not* thread safe to begin with before this PR, so we're not necessarily violating any contracts with users of the class. That being said, it may still be worth it to consider this, and update the classes in the `multi` package accordingly!